blazor/datagrid/how-to/customize-filter-popup-using-css.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: Customize filter popup using CSS in Blazor DataGrid | Syncfusion
+description: Learn here all about Customizing filter popup using CSS in Syncfusion Blazor DataGrid component and more.
+platform: Blazor
+control: DataGrid
+documentation: ug
+---
+
+# Customize Filter Popup Using CSS
+
+You can set the height and width for each column in the filter popup dialog using the CSS style.
```

Comment:
You can set the height and width of each column in the filter popup dialog using the CSS style.